### PR TITLE
Problem 2 for js1

### PR DESCRIPTION
**Pull request by rishikadall**

### CODE REVIEW

File: curriculum/js1/2.js [hunk 1]
```diff
@@ -6,9 +6,8 @@
  * @returns {number}
  */
 
-const solution = (a, b, c) => {
-  return 0;
-};
+const solution = (a, b, c) => { if (a > b && a > c) return a; else if (b > a && b > c) return b; else return c}
```
Comment (michaelbayday) [inline curriculum/js1/2.js]: ``const solution = (a,b,c) => {
if (a > b && a > c) {
return a;
} else if (b > a && b > c) {
return b;
}
return c;
}

Comment (michaelbayday) [inline curriculum/js1/2.js]:
```javascript
+const solution = (a, b, c) => { if (a > b && a > c) return a; else if (b > a && b > c) return b; else return c}
```

Comment (songz) [inline curriculum/js1/2.js]: can you split them into multiple lines?

Comment (songz) [reply curriculum/js1/2.js]: looks really messy.

Comment (rishikadall) [reply curriculum/js1/2.js]: done

**Comment by rishikadall** [comment]
merging to wrong branch

**Pull request closed without merging**